Sources/Defaults/Defaults+Bridge.swift

```diff
@@ -407,15 +412,45 @@ extension Defaults {
 				return nil
 			}
 
-			return NativeColor.bridge.serialize(NativeColor(value))
+			guard
+				let cgColor = value.cgColor,
+				let colorSpace = cgColor.colorSpace?.name as? String,
+				let components = cgColor.components
+			else {
+				return NativeColor.bridge.serialize(NativeColor(value))
+			}
+
+			return [colorSpace, components]
 		}
 
 		public func deserialize(_ object: Serializable?) -> Value? {
-			guard let nativeColor = NativeColor.bridge.deserialize(object) else {
+			if let object = object as? NativeColor.Serializable {
+				guard let nativeColor = NativeColor.bridge.deserialize(object) else {
+					return nil
+				}
+
+				return Value(nativeColor)
+			}
+
+			guard
+				let object = object as? [Any],
+				let rawColorspace = object[0] as? String,
+				let colorspace = CGColorSpace(name: rawColorspace as CFString),
+				let components = object[1] as? [CGFloat],
+				let cgColor = CGColor(colorSpace: colorspace, components: components)
+			else {
+				return nil
+			}
+
+			#if os(macOS)
+			guard let nativeColor = NativeColor(cgColor: cgColor) else {
 				return nil
 			}
```

Comment:
This can be removed too. Just pass cgColor directly on macOS too.

Reply:
Many thanks for the reminder!
Getting confused between the availability of `init(_ cgColor:)` and `init(cgColor:)`:sweat_smile:.